File: tests/claude-plugin-integration/test-repos/php-sample/PaymentService.php
<?php

require_once 'User.php';

class PaymentService {
    public function processPayment(User $user, float $amount): bool {
        echo "Processing \${$amount} for {$user->name}\n";
        return true;
    }

    public function refundPayment(User $user, float $amount): bool {
        echo "Refunding \${$amount} to {$user->name}\n";
        return true;
    }
}

File: tests/claude-plugin-integration/test-repos/csharp-sample/PaymentService.cs
using System;

public class PaymentService
{
    public bool ProcessPayment(User user, double amount)
    {
        Console.WriteLine($"Processing ${amount:F2} for {user.Name}");
        return true;
    }

    public bool RefundPayment(User user, double amount)
    {
        Console.WriteLine($"Refunding ${amount:F2} to {user.Name}");
        return true;
    }
}
